packages/protocol/scripts/bash/invite.sh

```diff
@@ -26,10 +27,15 @@ if ! nc -z 127.0.0.1 8545 ; then
   exit 1
 fi
 
-yarn run build && \
-gcloud config set project celo-testnet
-gcloud kms decrypt --ciphertext-file=twilio-config.enc --plaintext-file=twilio-config.js \
-  --key=github-key --keyring=celo-keyring --location=global && \
+if [ "$FAST" == true ] ; then
+  echo "Fast mode is on, protocol won't be rebuild and twilio config won't be fetched from KMS."
```

Comment:
nit: "rebuilt", "won't be decrypted with KMS"

Reply:
Done.